gflownet.py

```diff
@@ -380,8 +392,13 @@ def get_training_data(self, batch_size):
             path_id,
             state_id,
         ) = zip(*batch)
-        
+
         rewards = self.env.get_reward(input_reward, done)
+        terminalStates = rewards[rewards!=0]
+        # mean of just the terminal states
+        self.logger.log_metric('rewards', np.mean(terminalStates))
```

Comment:
What is the point of logging the reward and the proxy, as we can deduce the other with reward2acq (here acq is the proxy value for single fidelity simplest case) ?